content/sensu-enterprise-dashboard/2.10/configuration.md

```diff
@@ -33,6 +33,92 @@ Enterprise Dashboard configuration.
 _NOTE: the Sensu Enterprise Dashboard requires two configuration scopes: `sensu`
 and `dashboard` (see [Dashboard definition specification][8], below)._
 
+For a more advanced configuration making use of RBAC and SSL, consider the example below:
+
+{
+  "sensu": [
+    {
+      "name": "Sensu Deployment 1",
+      "host": "localhost",
+      "port": 4567,
+      "timeout": 5
+    },
+    {
+      "name": "Sensu Deployment 2",
+      "host": "10.0.1.10",
+      "port": 4567,
+      "timeout": 5
+    }
+  ],
+  "dashboard": {
+    "host": "0.0.0.0",
+    "port": 3000,
+    "interval": 5,
+    "users": [
+      {
+        "username": "sensu_user_1",
+        "password": "{crypt}EXAMPLEPASSFORUSER1"
+      },
+      {
+        "username": "sensu_user_1",
+        "password": "{crypt}EXAMPLEPASSFORUSER2"
+      }
+    ],
+    "ssl": {
+      "certfile": "/etc/sensu/ssl/cert.pem",
+      "keyfile": "/etc/sensu/ssl/key.pem",
+      "ciphersuite": [
+        "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+        "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+        "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+        "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA",
+        "TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA",
+        "TLS_RSA_WITH_AES_128_GCM_SHA256",
+        "TLS_RSA_WITH_AES_256_GCM_SHA384",
+        "TLS_RSA_WITH_AES_128_CBC_SHA",
+        "TLS_RSA_WITH_AES_256_CBC_SHA"
+      ],
+      "tlsminversion": "tls12"
+     },
+    "usersOptions": {
+    "dateFormat": "YYYY-MM-DD HH:mm:ss",
+    "defaultTheme": "uchiwa-default",
+    "disableNoExpiration": false,
+    "favicon": "/etc/sensu/favicon.png",
+    "logoURL": "/etc/sensu/logo.jpg",
+    "requireSilencingReason": false,
+    "silenceDurations": [ 0.5, 2 ]
+    },
+    "github": {
+      "clientId": "GITHUBCLIENTID",
+      "clientSecret": "GITHUBCLIENTSECRET",
+      "server": "https://github.com",
+      "roles": [
+        {
+          "name": "admin",
+          "members": [
+             "my-github-org/admins"
+          ],
+          "datacenters": [],
+          "subscriptions": [],
+          "readonly": false,
+          "accessToken": "MYACCESSTOKEN"
+        }
+      ]
+     }
+   }
+}
+
+This example makes use of the following:
+
+* Multiple Sensu datacenters
+* Local users (for fallback in the event Github cannot be reached)
+* SSL (a full list of supperted ciphers can be found [here][golang-ssl])
+* Default options (which you can view under the [Uchiwa docmentation][uchiwa-options])
```

Comment:
Looks like you're missing the link references for thse

Reply:
Addressed in latest push.